include/mbedtls/rsa.h

```diff
@@ -877,9 +883,10 @@ int mbedtls_rsa_rsaes_oaep_decrypt( mbedtls_rsa_context *ctx,
  *                 return #MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED.
  *
  * \param ctx      The initialized RSA context to use.
- * \param f_rng    The RNG function. It is needed for PKCS#1 v2.1 encoding
- *                 and for \p mode set to #MBEDTLS_RSA_PRIVATE, and ignored
- *                 otherwise.
+ * \param f_rng    The RNG function. If the padding mode is PKCS#1 v2.1
```

Comment:
That's still not right I'm afraid. V2.1 needs an RNG regardless of the mode. For v1.5 the RNG is used for blinding if provided but is not necessary for functionality.